src/views/v2/components/icons/play.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Template for the play icon.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events-virtual/components/play.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link http://m.tri.be/1aiy
+ *
+ * @version TBD
+ *
+ * @var array $classes Additional classes to add to the svg icon.
+ */
+
+$svg_classes = [ 'tribe-common-c-svgicon', 'tribe-common-c-svgicon--play' ];
+
+if ( ! empty( $classes ) ) {
+	$svg_classes = array_merge( $svg_classes, $classes );
+}
+?>
+<svg <?php tribe_classes( $svg_classes ); ?> xmlns="http://www.w3.org/2000/svg" viewBox="0 0 10 12"><path d="M10 6a1 1 0 01-.47.848l-8 5a.996.996 0 01-1.237-.14A.999.999 0 010 11V1A1 1 0 011.53.153l8 5A1 1 0 0110 6z" fill="#FFF" fill-rule="nonzero"/></svg>
```

Comment:
let's remove this fill and set from CSS

Reply:
@paulmskim problem here is that the venue page (where the play icon is called) doesn't have a `tribe-common` class on the container (for some reason, not sure if it's intentional) therefore I had to move the `.tribe-common-c-svgicon--play path` definition outside `.tribe-common` in the `skeleton/_svgicon.pcss` file.

Not sure if that's ok or it's better to add the class to the container?

Reply:
the venue page? or the event single?

Reply:
the venue page doesn't have that class

Reply:
styles remove from common and moved to VE